### ISSUE TITLE
Tab Behavior

### ISSUE THREAD

**Issue by GoogleCodeExporter** [potatomushclient/potato#41]
```
Tabs in quantities of 13 or more go past the edge of the window (think like
a sword and the sheath).

My suggestion: Tabs similar to Chrome Browser.
Also: Remove the #'s, move the character names under the tab name (Or only
in the title of the client), Smaller font. Remove the circle, instead color
the tab differently.
```

Original issue reported on code.google.com by `ThinkWri...@gmail.com` on 14 Sep 2009 at 3:13
- Blocked on: #185
- Merged into: #240
